docs/UserGuide.md

```diff
@@ -193,20 +203,21 @@ Format: `edit CLIENT_INDEX MEETING_INDEX [n/NAME] [dt/DATE_TIME] [v/VENUE] [d/DE
 
 Examples:
 
-- `edit 1 2 n/Annual Review dt/2023-12-01 10:00 v/Main Office` Edits the name, date/time, and venue of the 2nd meeting for the 1st client.
-- `edit 2 1 n/Budget Meeting d/` Edits the name of the 1st meeting for the 2nd client and clears all existing descriptions.
+- `editMeeting clientIndex/1 meetingIndex/2 n/starbucks meeting dt/01-01-2024 12:00` Edits the description and date/time of the 1st meeting of the 1st client.<br>
+  ![edit meeting result](images/resultImages/editMeetingResult.png)
 
 ### Deleting a Meeting: `delete`
 
 Deletes a specific meeting for a client.
 
-Format: `delete CLIENT_INDEX MEETING_INDEX`
+Format: `deleteMeeting clientIndex/CLIENT_INDEX meetingIndex/MEETING_INDEX`
 
 - Deletes the meeting specified by `MEETING_INDEX` for the client specified by `CLIENT_INDEX`. Both indexes must be positive integers 1, 2, 3, …​.
 
 Example:
 
-- `delete 3 1` Deletes the first meeting for the third client.
+- `deleteMeeting clientIndex/2 meetingIndex/1` Deletes the first meeting for the first client.<br>
+  ![delete meeting result](images/resultImages/deleteMeetingResult.png)
```

Comment:
I like how you add the descriptive images to better inform users on what a successful execution of the command would look like!